setup.py

```diff
@@ -78,6 +78,12 @@
     packages=['markdown', 'markdown.extensions'],
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
     install_requires=['setuptools >= 36'],
+    extras_require={
+        'testing': [
+            'coverage<5.0',
```

Comment:
We previously were using `coverage<4.0` but this is `coverage<5.0`. We added the previous restriction (in 20b17c29) because  Coverage 4.0 dropped support for PY3.2.  However, we have also dropped support for PY3.2 (in d7764996). That being the case, we don't need that restriction. Currently, Coverage is at version 4.5.2, so there is no 5.0. Maybe we should remove the version restriction altogether?

Reply:
I also note that Coverage currently supports PY2.6 and Py3.3, neither of which we support. That being the case, I see no need to have a version restriction here.

Reply:
Cool, I've just added/kept it since coverage.py is in alpha already and quite a bit different (but should be compatible).  So this is good to remove.